src/functionalTest/java/uk/gov/hmcts/divorce/testutil/FunctionalTestSuite.java

```diff
@@ -50,13 +50,13 @@
     @Autowired
     private CoreCaseDataApi coreCaseDataApi;
 
-    protected CaseData caseData() {
-        return CaseData.builder()
-            .petitionerFirstName(TEST_FIRST_NAME)
-            .petitionerLastName(TEST_LAST_NAME)
-            .divorceOrDissolution(DIVORCE)
-            .petitionerEmail(TEST_USER_EMAIL)
-            .build();
+    protected Map<String, Object> caseData() {
+        Map<String, Object> caseDataMap = new HashMap<>();
+        caseDataMap.put("petitionerFirstName", TEST_FIRST_NAME);
+        caseDataMap.put("petitionerLastName", TEST_LAST_NAME);
+        caseDataMap.put("divorceOrDissolution", DIVORCE);
+        caseDataMap.put("petitionerEmail", TEST_USER_EMAIL);
```

Comment:
would be nice to use constants so you can refer everything from one place and if field name changes it reflects everywhere

Reply:
👍